Support.md - how to take Screenshots (#263)
Manjaro-Sway/manjaro-sway: SUPPORT.md

```
set $background /usr/share/backgrounds/whatever/file/you/like.png
```

> ### How can I take a ScreenShot  (Easy way)

Reviewer: ```suggestion
### How can I take a Screenshot?
```

> it's very easy to take a full screenshot on Manjaro Sway as the process for this is little different than other out of the box 

Reviewer: ```suggestion
```

> ```

Reviewer: ```suggestion
```

> Press printscreen button on your keyboard , after that you will see some options on waybar , just press - Shift +o (small o) in your keyborad together and you will see an screenshot hasbeen taken by Manjaro Sway

Reviewer: ```suggestion

Press the print-screen button on your keyboard and you will be presented with some options on waybar. Depending on what you would like to take a screenshot of, you have to press a combination of keys. To take a full screenshot of the current screen just press - `Shift + o` on your keyboard.
```

> Enjoy your Manjaro :)

Reviewer: ```suggestion
```

> ```

Reviewer: ```suggestion
```

```
set $background /usr/share/backgrounds/whatever/file/you/like.png
```

### How can I take a Screenshot?

> Press the print-screen button on your keyboard and you will be presented with some options on the waybar. Depending on what you would like to take a screenshot of, you have to press a combination of keys. To take a full screenshot of the current screen just press - `Shift + o` on your keyboard.

Reviewer: ```suggestion
Press the `Print` button on your keyboard and you will be presented with some options on the waybar. Depending on what you would like to take a screenshot of, you have to press a combination of keys. To take a full screenshot of the current screen just press - `Shift + o` on your keyboard.
```

```
set $background /usr/share/backgrounds/whatever/file/you/like.png
```

### How can I take a Screenshot?

Press the print-screen button on your keyboard and you will be presented with some options on the waybar. Depending on what you would like to take a screenshot of, you have to press a combination of keys. To take a full screenshot of the current screen just press - `Shift + o` on your keyboard.

> 

Reviewer: ```suggestion
```
